packages/vector_graphics_compiler/pubspec.yaml

```diff
@@ -14,6 +14,7 @@ dependencies:
   meta: ^1.7.0
   path_parsing: ^1.0.0
   xml: ^5.3.1
+  args: ^2.3.0
```

Comment:
Would it make sense to have this be a dev dependency?  Does that even work?

Ideally it'd only be a dependency for the files in bin.

Reply:
Oh, you're right this can be a dev_dependency

Reply:
fixed!